libs/designer-ui/src/lib/querybuilder/Group.tsx

```diff
@@ -204,19 +207,19 @@ export const Group = ({
     }
   };
 
-  const handleMoveChild = (childIndex: number, moveOption: MoveOption) => {
-    if (childIndex <= 0 && moveOption === MoveOption.UP) {
-      // const newItems = { ...groupProps };
-    } else if (childIndex >= groupProps.items.length - 1 && moveOption === MoveOption.DOWN) {
-      // come back
-    } else {
-      const newItems = { ...groupProps };
-      const child = newItems.items[childIndex];
-      newItems.items[childIndex] = newItems.items[childIndex + (moveOption === MoveOption.UP ? -1 : 1)];
-      newItems.items[childIndex + (moveOption === MoveOption.UP ? -1 : 1)] = child;
-      handleUpdateParent(newItems, index);
-    }
-  };
+  // const handleMoveChild = (childIndex: number, moveOption: MoveOption) => {
+  //   if (childIndex <= 0 && moveOption === MoveOption.UP) {
+  //     // const newItems = { ...groupProps };
+  //   } else if (childIndex >= groupProps.items.length - 1 && moveOption === MoveOption.DOWN) {
+  //     // come back
+  //   } else {
+  //     const newItems = { ...groupProps };
+  //     const child = newItems.items[childIndex];
+  //     newItems.items[childIndex] = newItems.items[childIndex + (moveOption === MoveOption.UP ? -1 : 1)];
+  //     newItems.items[childIndex + (moveOption === MoveOption.UP ? -1 : 1)] = child;
+  //     handleUpdateParent(newItems, index);
+  //   }
+  // };
```

Comment:
Are these and the other commented code in this file not needed anymore?

Reply:
This logic was in part trying to emulate the original Query builder from an external library, I forgot exactly what issue I ran into, but because at that point we were rushing to public preview I was planning on revisiting it in the future - commented out because this code is still unused, but I do hope to come back to it (I'll create an issue to track it)

Reply:
btw this was a purely cosmetic miss, which was why it was not prioritized https://github.com/Azure/LogicAppsUX/issues/3796 - here is issue for tracking